Sources/App/Collections/VanityCollection.swift

```diff
@@ -148,8 +148,10 @@ final class VanityCollection: RouteCollection {
                     print("APNS error: \(error)")
                 }
             })
-
-            return Response(status: .seeOther, headers: [.location: String(describing: request.uri)])
+            
+            return Response(status: .seeOther,
```

Comment:
Is this not an `EmptyResponse` as well ? (beside the fact that it already have the content type).

I would have extended the `Response()` function and override `headers.contentType` if it does not exist, for all the responses, not just the one with no `headers/contentType`.